docs/administration-guide/databases/oracle.md

```diff
@@ -17,6 +17,48 @@ Before downloading this JAR you may need to sign up for a free account with Orac
 Metabase will automatically make the Oracle driver available if it finds the Oracle JDBC driver JAR in the Metabase plugins directory when it starts up.
 All you need to do is create the directory, move the JAR you just downloaded into it, and restart Metabase.
 
+### Connecting with SSL
+
+To connect to Oracle via SSL, check the `Use a secure connection (SSL)?` option on the connection setup page.  This will
+enable encryption. Other SSL features (including client and/or server authentication) can be added as follows.
+
+#### Server authentication
+
+To configure the client (Metabase) to authenticate the identity of the server (the Oracle server), you may need to
+configure a trust store file that includes the server's root CA, so that the JVM running Metabase trusts its
+certificate chain. Refer to the Oracle documentation
+[here](https://docs.oracle.com/javase/8/docs/technotes/tools/unix/keytool.html) on using `keytool` to manage key
+and trust store files, importing certificates, etc. Once you have a trust store file ready, add the following JVM
+options for Metabase to use it:
+
+```
+-Djavax.net.ssl.trustStore=/path/to/truststore.jks
+-Djavax.net.ssl.trustStoreType=JKS \
+-Djavax.net.ssl.trustStorePassword=<trustStorePassword>
+```
+
+With this done, the SSL connection to Oracle will authenticate the server.
+
+For more information on setting up a trust store for AWS RDS Oracle instances, see the instructions provided by Amazon
+[here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.Oracle.Options.SSL.html#Appendix.Oracle.Options.SSL.JDBC).
+Note that if you require connecting to other databases using SSL, you will likely want to add the RDS CA to your
+existing trust store file (likely called `cacerts`), instead of creating a new one, as shown in those examples.
+
+#### Client authentication
+
+To configure the server (the Oracle server) to authenticate the identity of the client (Metabase), you need to
+configure a key store file that includes the client's private key. The steps are almost identical to those
+above, except that you will import the client's private key into the keystore rather than a root CA into a truststore
+file.
```

Comment:
```suggestion
To configure the server (the Oracle server) to authenticate the identity of the client (Metabase), you need to
configure a keystore file that includes the client's private key. The steps are almost identical to those
above, except that you will import the client's private key into the keystore rather than a root CA into a truststore
file.
```
Amazon spells it keystore, I've also seen it as KeyStore.

Reply:
Done in new commit (also changed "trust store" to "truststore" for consistency).